test/absurd.lean

```diff
@@ -0,0 +1,48 @@
+import Std.Tactic.Basic
+import Std.Tactic.GuardMsgs
+
+/-! Tests for `absurd` tactic -/
+
+-- Basic example
+/--
+error: unsolved goals
+p : Prop
+h : p
+⊢ ¬p
+-/
+#guard_msgs in
+example {p : Prop} (h : p) : False := by
```

Comment:
you should use goals that are not just `False` in the tests

Reply:
Any preference? Random terms? Or `sorry`?

Reply:
Just anything else - you can't tell from this test whether the tactic requires that the goal is `False`